hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/src/main/webapp/app/utils/converter.js

```diff
@@ -83,7 +83,7 @@ export default {
     return total * 1000;
   },
   timeStampToDate: function(timeStamp) {
-    return convertTimestampWithTz(timeStamp, "YYYY/MM/DD HH:mm:ss");
+    return timeStamp == 0 ? "NA" : (0, _yarnUiUtilsDateUtils.convertTimestampWithTz)(timeStamp, "YYYY/MM/DD HH:mm:ss");
```

Comment:
jshint:24:W116:Expected '===' and instead saw '=='.
jshint:40:W117:'_yarnUiUtilsDateUtils' is not defined.
jshint:85:W067:Bad invocation.